README.md

```diff
@@ -181,6 +181,15 @@ See [an example pom in our Spring Boot application](e2e-test-app/pom.xml)
 
 ### Simple chat completion
 
+```java
+final OpenAiChatCompletionOutput result =
+    OpenAiClient.forModel(GPT_35_TURBO).chatCompletion("Hello World! Why is this phrase so famous?");
```

Comment:
I like the simplicity of the API, PP ready 😉 

We could consider to offer an optional:

```suggestion
    OpenAiClient.forModel(GPT_35_TURBO)
        .withSystemPrompt("You are a helpful AI")
        .chatCompletion("Hello World! Why is this phrase so famous?");
```

I think it would be useful to have a fixed system prompt for multiple requests, as the system prompt is usually less dynamic than the user prompt.